packages/xstate-react/src/useInterpret.ts

```diff
@@ -36,18 +35,15 @@ function toObserver<T>(
   };
 }
 
-export function useInterpret<
+export function useIdleInterpreter<
```

Comment:
Not the best name... but naming things is hard™ and naming some hooks is even harder 😅